src/models/config.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * SPDX-FileCopyrightText: 2021 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+import { loadState } from '@nextcloud/initial-state'
+
+const alwaysShowViewer = loadState(appName, 'always_show_viewer', false)
```

Comment:
we should consider adding a type or an interface to `loadState` to use Typescript properly

```suggestions
const alwaysShowViewer = loadState<boolean>(appName, 'always_show_viewer', false)
```

references
https://github.com/IONOS-Productivity/nc-server/blob/master/apps/settings/src/store/apps-store.ts#L25
https://github.com/IONOS-Productivity/nc-server/blob/master/apps/files/src/store/userconfig.ts#L13